app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailFragment.kt

```diff
@@ -34,6 +39,9 @@ class SessionDetailFragment : Fragment(), Injectable {
         ViewModelProviders.of(activity!!, viewModelFactory).get(SessionDetailViewModel::class.java)
     }
 
+    val speakerSummary: SpeakersSummaryLayout
```

Comment:
[nits] Do you use this property?

Reply:
Thanks! I forgot to remove this property..., I will remove on next commit.